versions/bukkit.json

```diff
@@ -1,5 +1,6 @@
 {
     "versions": [
+        "1.16.1",
```

Comment:
[SpigotMC doesn't distribute Bukkit versions higher than 1.15.2](https://www.spigotmc.org/threads/spigot-bungeecord-1-16-1.447405/#post-3852349), you have to remove that line. This file's data is basically frozen now.

Reply:
Fair point. Although it is still possible to develop Bukkit only plugins even if SpigotMC does not distribute it. Just need to run BuildTools locally to get that dependency. I will be removing it on the  grounds of Bukkit only plugins not being supported by the maintainer SpigotMC. 
Not that anyone should have been making Bukkit only plugins before this....

Reply:
Well, the thing is, the plugin configures the build tools to use the SpigotMC maven repository, and since they don't put Bukkit 1.16.1 there anymore we can't use it as a version, it will fail during dependency resolution because it doesn't exist.

Reply:
That seems logical. I have removed it. And marking this resolved